src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -45,15 +47,16 @@
 			public static final int ballioElevatorAndIntakeRoller = 3;
 			public static final int climbMotorA = 4;
 			public static final int climbMotorB = 5;
-			public static final int flywheelLeftMotor = 6; // WIP
-			public static final int flywheelRightMotor = 7; // WIP
+			public static final int flywheelMotorA = 6; // WIP
```

Comment:
The two motors are treated identically in the code, so the distinction doesn't really make sense (just like the two motors for the left wheel). I would be OK with labelling them physically as A and B, though.